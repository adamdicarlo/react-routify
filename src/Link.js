import React from 'react'

const Link = ({to, ...rest}) => {
  const props = {
    href: to,
    ...rest
  }

  return <a {...props} />
}

export default Link
