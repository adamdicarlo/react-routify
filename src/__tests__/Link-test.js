import React from 'react'
import renderer from 'react-test-renderer'
import Link from '../Link'

test('Link should render', () => {
  const link = <Link to="/nowhere" className="some-class" id="something" />
  const component = renderer.create(link)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
