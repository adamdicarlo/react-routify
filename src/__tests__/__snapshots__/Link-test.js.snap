exports[`test Link should render 1`] = `
<a
  className="some-class"
  href="/nowhere"
  id="something" />
`;
